src/constructs/loadbalancing/clb.ts

```diff
@@ -0,0 +1,32 @@
+import type { CfnLoadBalancer, LoadBalancerProps } from "@aws-cdk/aws-elasticloadbalancing";
+import { LoadBalancer } from "@aws-cdk/aws-elasticloadbalancing";
+import type { GuStack } from "../core";
+
+enum DeletionOverrideProperties {
```

Comment:
I'm not sure how much of the `cdk` jargon we're hoping to hide from users of this library, but `RemovableElbProperties` makes more sense to someone (like me!) who hasn't spent much time looking at `cdk` yet!